manifests/c/Cygwin/Cygwin/3.3.4/Cygwin.Cygwin.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Cygwin.Cygwin
+PackageVersion: 3.3.4
```

```suggestion
PackageVersion: 3.3.5
```
Output shows that it installs 3.3.5? You will also have to update the package version folder from `3.3.4` to `3.3.5`
```
Augmented Transaction List:
   [...]
   1 install cygwin          3.3.5-1
   [...]
```